### PR TITLE
fix bugs - helper text copy ingress ip & edit preinjected file & prob…

### PR DESCRIPTION
**Pull request by GonGH-E**
…es fields validate

### CODE REVIEW

File: frontend/src/forms/ComponentLike/Probes.tsx [hunk 1]
```diff
@@ -225,7 +222,6 @@ class RenderProbe extends React.PureComponent<Props> {
             <Field
               name={`${name}.tcpSocket.port`}
               component={RenderNestedTextfield}
-              validate={ValidatorRequired}
```
Comment (davidqhr) [inline frontend/src/forms/ComponentLike/Probes.tsx:228]: Port is a required field.

Comment (davidqhr) [reply frontend/src/forms/ComponentLike/Probes.tsx:228]: You should check each field carefully.

**Review by davidqhr — CHANGES_REQUESTED** [review]

File: frontend/src/forms/ComponentLike/Probes.tsx [hunk 2]
```diff
@@ -128,14 +127,12 @@ class RenderProbe extends React.PureComponent<Props> {
               component={RenderNestedTextfield}
               placeholder="8080"
               parse={NormalizePositiveNumber}
-              validate={ValidatorRequired}
               style={{ width: 60 }}
             />
             <Field
               name={`${name}.httpGet.path`}
               component={RenderNestedTextfield}
               placeholder="/healthy"
```
Comment (davidqhr) [inline frontend/src/forms/ComponentLike/Probes.tsx:135]: This field need a parse attribute. If it's not blank, it should have prefix "/".

File: frontend/src/forms/ComponentLike/Probes.tsx [hunk 3]
```diff
@@ -119,7 +119,6 @@ class RenderProbe extends React.PureComponent<Props> {
               name={`${name}.httpGet.host`}
               component={RenderNestedTextfield}
               placeholder="0.0.0.0"
-              validate={validateHostWithWildcardPrefix}
```
Comment (davidqhr) [inline frontend/src/forms/ComponentLike/Probes.tsx:122]: Why delete this validator? Even this field is optional, you should also consider to validate context when it's not empty.

**Comment by GonGH-E** [comment]
@davidqhr

**Pull request closed without merging**